### ISSUE TITLE
invalid return values as list

### ISSUE THREAD

**Issue by tam17aki** [blink1073/oct2py#159]
I have prepared the following m-file.
```
function [y, z] = test(x)
       y = x + 1;
       z = x * 2;
end
```

Then I tested the m-file as:
```
from oct2py import octave
import numpy as np
x = np.array([[1, 2], [3, 4]], dtype=float)
y, z = octave.test(x)
print(y)
print(z)
```
Unfortunately, the following results were obtained:
`[2. 3.]
 [4. 5.]
`

I think that expected results should be
```
[[2. 3.]
 [4. 5.]]
[[2. 4.]
 [6. 8.]]
```
The latest version of oct2py cannot return the MATLAB list and cannot separate it into arrays, is it correct ?
I am sorry if i have misunderstood.

Thank you.

**Comment by tam17aki** [comment]
Additionaly, the following code raise an error:
```
from oct2py import octave
x = 1.0
y, z = octave.test(x)
print(y)
print(z)
```
```
Traceback (most recent call last):
  File "test.py", line 3, in <module>
    y, z = octave.test(x)
TypeError: cannot unpack non-iterable float object
```

**Comment by blink1073** [comment]
Hi @tam17aki, oct2py no longer infers the number of output arguments (it was too brittle).  You can specify using `nout=2`, e.g. `octave.test(x, nout=2)`.

**Comment by tam17aki** [comment]
Hi @blink1073,

Thank you very much!
Now I understand the specification using nout,  and I can obtain the expected results.

Best regards,
Akira

**Comment by tam17aki** [comment]
I suggest that the official example document should be fixed about `nout = 2`, because it says 
`out, oclass = octave.roundtrip(x)`
http://blink1073.github.io/oct2py/source/examples.html

Best regards,
Akira

**Comment by yasirroni** [comment]
Solved in [f16f0bd](https://github.com/blink1073/oct2py/pull/170/commits/f16f0bd36c650a4f813790b2c48f3436031daf1a)
